src/EventListener/FormListener.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Bolt\EventListener;
```

Comment:
I know EventSubscribers have nothing *specifically* to do with Kernel events, but currently everything in this namespace does. Maybe this would be less confusing in `Bolt\Form\EventListeners`?